managementAgent.js

```diff
@@ -3,8 +3,16 @@ const Uuid = require('uuid');
 
 const logger = require('./lib/utilities/logger');
 const { initManagement } = require('./lib/management');
+const { addConfigurationListener } = require('./lib/management/configuration');
 const _config = require('./lib/Config').config;
+const WebSocket = require('ws');
+
+
+//TODO: signal handler to cleanly shutdown server
+//TODO: ping messages between server and client
```

Comment:
I think it would be good to implement the ping part in this PR

Reply:
done (I missed this comment)